src/openzaak/conf/includes/base.py

```diff
@@ -97,6 +97,12 @@
     + PLUGIN_INSTALLED_APPS
 )
 
+# `django.contrib.sites` added at the project level because it has been removed at the packages level.
+# This component is deprecated and should be completely removed.
+# To determine the project's domain, use the `SITE_DOMAIN` environment variable.
+if "django.contrib.sites" not in INSTALLED_APPS:
+    INSTALLED_APPS.append("django.contrib.sites")
```

Comment:
Can you also add back the SITE_ID envvar here? This setting is required if sites is used https://github.com/maykinmedia/open-api-framework/pull/121/files#diff-8b7db4fa63ef0f1e64c48e8cf49d8beacf93587629abd42f1f6f9cd9a0aa8f06L32